app/helpers/participants_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ParticipantsHelper
   def hall_of_fame_title
-    params[:action] == "hall_of_fame" ? "Hall Of Fame" : "Organizer Of Fame"
+    params[:action] == "hall_of_fame" ? "HALL OF FAME_" : "ORGANIZER OF FAME_"
```

Comment:
Just leave it as Hall Of Fame. You really don't need to have all of `pt.yml` in portuguese. It's just that it's the file selected for PT users. It can still contain english.